### PR TITLE
EZP-30344: Language limitation

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-30344
| Versions      | **master (2.5.1)**

To do:
- [x] Verify after the code PR is merged

### CODE REVIEW

File: docs/guide/limitations.md [hunk 1]
```diff
@@ -379,6 +380,30 @@ A Limitation to specify if the User has access to Content within a specific Subt
 |------|------|------|
 |`<Location_pathString>`|`<Location_name>`|All valid location `pathStrings` can be set as value(s)|
 
+### TranslationLimitation
+
+A Limitation to specify if the User has access to work on the specified translation.
+
+A user with this Limitation is allowed to:
+- create new content with the given translation(s) only,
```
Comment (michalmilc90) [inline docs/guide/limitations.md]:
```suggestion
- create new content with the given translation(s) only
```

File: docs/guide/limitations.md [hunk 2]
```diff
@@ -379,6 +380,30 @@ A Limitation to specify if the User has access to Content within a specific Subt
...
+- edit content by adding a new translation or modifying an existing translation,
```
Comment (michalmilc90) [inline docs/guide/limitations.md]:
```suggestion
- edit content by adding a new translation or modifying an existing translation
```

File: docs/guide/limitations.md [hunk 3]
```diff
@@ -379,6 +380,30 @@ A Limitation to specify if the User has access to Content within a specific Subt
...
+- publish content only when it results in adding or modifying an allowed translation,
```
Comment (michalmilc90) [inline docs/guide/limitations.md]:
```suggestion
- publish content only when it results in adding or modifying an allowed translation
```

File: docs/guide/limitations.md [hunk 4]
```diff
@@ -379,6 +380,30 @@ A Limitation to specify if the User has access to Content within a specific Subt
...
+- delete a translation based on a list of allowed translations.
```
Comment (michalmilc90) [inline docs/guide/limitations.md]:
```suggestion
- delete a translation based on a list of allowed translations
```

File: docs/guide/limitations.md [hunk 5]
```diff
@@ -44,8 +44,9 @@ If a function is absent from the tables below, it means that no Limitations can
 |read|true|true|true|true|true|true|-|State|
 |diff|true|true|true|true|true|-|-|-|
 |view_embed|true|true|true|true|true|-|-|-|
-|create|true|true|-|true|true|-|true|Owner of Parent</br>Content Type Group of Parent</br>Content Type of Parent</br>Parent Depth|
-|edit|true|true|true|true|true|true|true|State</br>WorkflowStage|
+|create|true|true|-|true|true|-|true|Owner of Parent</br>Content Type Group of Parent</br>Content Type of Parent</br>Parent Depth</br>Translation|
```
Comment (alongosz) [inline docs/guide/limitations.md]: I'm actually quite lost when it comes to this last column. Is it done like this because of space on a screen?

Comment (MagdalenaZuba) [reply docs/guide/limitations.md]: Yes, it is.

File: docs/guide/limitations.md [hunk 6]
```diff
@@ -44,8 +44,9 @@ If a function is absent from the tables below, it means that no Limitations can
 |read|true|true|true|true|true|true|-|State|
 |diff|true|true|true|true|true|-|-|-|
 |view_embed|true|true|true|true|true|-|-|-|
-|create|true|true|-|true|true|-|true|Owner of Parent</br>Content Type Group of Parent</br>Content Type of Parent</br>Parent Depth|
-|edit|true|true|true|true|true|true|true|State</br>WorkflowStage|
+|create|true|true|-|true|true|-|true|Owner of Parent, Content Type Group of Parent, Content Type of Parent, Parent Depth, Translation|
```
Comment (alongosz) [inline docs/guide/limitations.md]: Let's swap `Language` with `Translation`, so `Translation` is more visible. We could even get rid of `Language`.

File: docs/guide/limitations.md [hunk 7]
```diff
@@ -135,7 +136,14 @@ A Limitation to specify if the User has access to Content with a specific Conten
 
 #### LanguageLimitation
 
-A Limitation to specify if the User has access to Content in a specific language.
+A Limitation to specify if the User has access to work on the specified translation.
+
+A user with this Limitation is allowed to:
+
+- create new content with the given translation(s) only.
```
Comment (michalmilc90) [inline docs/guide/limitations.md]:
```suggestion
- Create new content with the given translation(s) only.
```

File: docs/guide/limitations.md [hunk 8]
```diff
@@ -135,7 +136,14 @@ A Limitation to specify if the User has access to Content with a specific Conten
...
+This only applies to creating the first version of a Content item
```
Comment (michalmilc90) [inline docs/guide/limitations.md]:
```suggestion
This only applies to creating the first version of a Content item.
```

File: docs/guide/limitations.md [hunk 9]
```diff
@@ -135,7 +136,14 @@ A Limitation to specify if the User has access to Content with a specific Conten
...
+- edit content by adding a new translation or modifying an existing translation
```
Comment (michalmilc90) [inline docs/guide/limitations.md]:
```suggestion
- Edit content by adding a new translation or modifying an existing translation.
```

File: docs/guide/limitations.md [hunk 10]
```diff
@@ -135,7 +136,14 @@ A Limitation to specify if the User has access to Content with a specific Conten
...
+- publish content only when it results in adding or modifying an allowed translation
```
Comment (michalmilc90) [inline docs/guide/limitations.md]:
```suggestion
- Publish content only when it results in adding or modifying an allowed translation.
```

**Pull request merged**